### ISSUE TITLE
This isn't an Issue, But Rather a Request

### ISSUE THREAD

**Issue by captainbob32** [viktormax3/Wii-banners#1]
Is it Alright if You Do More? Like, For Example:
-Rayman Raving Rabbids
-Rayman Raving Rabbids (Rev 1/Rev 2)
-Rayman Raving Rabbids 2
-Rayman Raving Rabbids: TV Party
-Super Mario Galaxy (Original)
-WarioWare Smooth Moves
and Raving Rabbids: Travel in Time?

Because I REALLY Like This Repository

**Comment by viktormax3** [comment]
Yes this is very possible i left it a bit abandoned i was intending to do this with my favorite games it was a very manual process but now there are new tools i think a little support i can do it

**Comment by ghost** [comment]
Ok

**Comment by RibShark** [comment]
What tools did you use for this? Want to edit some banners myself and I can't find anything that outputs .rlyt files.

**Comment by viktormax3** [comment]
I created it manually, there is no tool (as far as I know) that does this procedure automatically, the procedure was: 

1. Make a dump of the game, and unzip it (there are many tools that do this)
2. Unzip the "banner.bnr" (There are several very old tutorials)
3. Convert brlan and brylt files to xml, there are some new tools
4. Use some programs to view the internal workings of files, there are brylt and brlan file viewers, you can search the  brylt or brlan wiki tockdom on Google
5. You need the Wii SDK, specifically the animation editor (I don't remember its name)
6. I remember using a zip called Wii banner tools, there are newer programs to unzip u8 or ar files
